client/layout/logged-out.jsx

```diff
@@ -22,7 +22,8 @@ const LayoutLoggedOut = ( {
 		[ 'is-group-' + section.group ]: !! section,
 		[ 'is-section-' + section.name ]: !! section,
 		'focus-content': true,
-		'has-no-sidebar': true // Logged-out never has a sidebar
+		'has-no-sidebar': true, // Logged-out never has a sidebar
+		'wp-singletree-layout': primary,
```

Comment:
Mm, was just about to change this, then I thought—tidier in what sense?. `classnames` only cares about whether the arg is falsy or not. Would seem to be a couple of unnecessary negations.